harvard-huit/Harvard-Geospatial-Library
# Upgrading Puma to 5.4.0.

**Upgrading Puma to 5.4.0.**
* * *

**JIRA Ticket**: [LTSDEVS-6](https://jira.huit.harvard.edu/browse/LTSDEVS-6)

* **_Bonus! This is my first code commit / pull request from my new MacBook!_**

# What does this Pull Request do?
This upgrades Puma to 5.4.0. There is a high-severity upgrade for Puma. See the [Dependabot Alert](https://github.com/harvard-huit/Harvard-Geospatial-Library/security/dependabot/Gemfile.lock/puma/open).

# How should this be tested?

* Completely rebuild the container: `docker-compose -f docker-compose-local.yml up -d --build --force-recreate`
* Confirm that the bundler installs Puma 5.4.0
* Confirm HGL is still up and operational.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz @mferrarini @dl-maura
